core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/migration/MultiregionalBucketMigrationActor.scala

```diff
@@ -939,73 +939,93 @@ object MultiregionalBucketMigrationActor {
       // Transfer operations are listed after they've been started. For bucket-to-bucket transfers
       // we expect at most one operation. Polling the latest operation will allow for jobs to be
       // restarted in the cloud console.
-      outcome <- liftF {
+      operation <- liftF {
```

Comment:
Before, we were only updating the db when the transfer job had completed. This doesn't work if we want transfer stats while the job is in progress, so I've reworked this to support that